asserts/database.go

```diff
@@ -70,6 +83,8 @@ type DatabaseConfig struct {
 	// backstore for assertions, falls back to a filesystem based backstrore
 	// if not set
 	Backstore Backstore
+	// manager/backstore for keypairs, falls back to a filesystem based manager
```

Comment:
The same observations we've made on top of Backstore are valid here: it'd be better to make these explicit, and get rid of the Path attribute above. Per prior agreement, also fine to do on a follow up.